contracts/basic-bandwidth-generation/contracts/BandwidthGenerator.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity 0.6.6;
+
+import "./CosmosToken.sol";
+import "./Gravity.sol";
+
+/** 
+ * @title BandwidthGenerator
+ * @dev   Contract for generating basic bandwidth credentials on the Nym cosmos blockchain, 
+ *        using ERC20 representations of CW20 NYM as payment. 
```

Comment:
The nym token is native cosmos, not a CW20 token, so you can remove CW20 from here.